ci/federalist-pipeline.yml

```diff
@@ -0,0 +1,235 @@
+            channel: ((slack-channel))
+            username: ((slack-username))
+            icon_url: ((slack-icon-url))
+
+  - name: deploy
+    plan:
+      - get: src
+        resource: src-production
+        trigger: true
+        params: {depth: 1}
+        passed: [test-production]
+      - put: build-container-image
+        params:
+          build: src
+      - task: deploy
+        config:
+          <<: *cf-image
+          inputs:
+            - name: src
+            - name: build-container-image
+          run:
+            dir: src
+            path: ci/tasks/deploy.sh
+        params:
+          <<: *env-cf
+          CF_APP_NAME: federalist-build-container-production
+          CF_MANIFEST: .cloudgov/manifest.yml
+          CF_VARS_FILE: .cloudgov/vars/production.yml
+          CF_DOCKER_IMAGE_DIGEST: ../build-container-image/digest
+          CF_DOCKER_IMAGE_REPOSITORY: ((image-repository))
+          CF_DOCKER_USERNAME: ((docker-username))
+          CF_DOCKER_PASSWORD: ((docker-password))
+    on_failure:
+      in_parallel:
+        - put: gh-status
+          inputs: [src]
+          params: {state: failure}
+        - put: slack
+          params:
+            text: |
+              :x: FAILED: federalist build container deployment on production
+              <$ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|View build details>
+            channel: ((slack-channel))
+            username: ((slack-username))
+            icon_url: ((slack-icon-url))
+
+  - name: nightly-tasks-production
+    plan:
+      - get: src
+        resource: src-production
+        passed: [set-pipeline]
+        params: {depth: 1}
+      - get: nightly
+        trigger: true
+      - task: restage
+        config:
+          <<: *cf-image
+          inputs: [name: src]
+          run:
+            dir: src
+            path: ci/tasks/restage.sh
+        params:
+          <<: *env-cf
+          CF_APP_NAME: federalist-build-container-production
+
+############################
+#  RESOURCES
+
+resources:
+
+  - name: src-production
+    type: git
+    icon: github
+    source:
+      uri: ((git-base-url))/((garden-build-repository-path))
+      branch: main
+
+  - name: pr-main
+    type: pull-request
+    check_every: 1m
+    source:
+      repository: ((garden-build-repository-path))
+      access_token: ((gh-access-token))
+      base_branch: main
+      disable_forks: true
+      ignore_drafts: false
+
+  - name: nightly
+    type: time
+    source:
+      start: 12:00 AM
+      stop: 1:00 AM
+      location: America/New_York
+
+  - name: slack
+    type: slack-notification
+    source:
+      url: ((slack-webhook-url))
+
+  - name: gh-status
+    type: cogito
+    check_every: 1h
+    source:
+      owner: 18F
+      repo: federalist-garden-build
```

Comment:
We may want to turn this into a credhub variable to make our GH org migration simpler.